example/example.js

```diff
@@ -0,0 +1,80 @@
+// Slow things down a little!
+function delay(time) {
+  return new Promise(resolve => setTimeout(resolve, time));
+}
+
+function showSimplex(id, obj) {
+    var points = obj.simplex().map(el => el.location);
```

Comment:
`simplex` returns an array, not just the current state? Oh, because it includes all the points in the simplex, not just the best one? So what should we be showing on the fit plot at that point? the solution for the first location in the simplex is always the best so show that?

Reply:
yes, first is always the best. But it's not that exciting to look at and I was curious how the simplex itself evolves. You can see it conforming to the shape of the surface, even though it does not explicitly have gradient information, which is cool